sections/all_map_locations.tex

```diff
@@ -8,11 +8,12 @@ \subsection*{Symbols on the Map}
 \begingroup
   \renewcommand{\arraystretch}{1.5}
   \newcolumntype{Y}{>{\centering\arraybackslash}X}
-  \begin{tabularx}{\linewidth}{Y m{0.7\linewidth}}
+  \begin{tabularx}{\linewidth}{Y m{0.8\linewidth}}
     \small
     \mbox{\textbf{I-VII}} & Field Difficulty corresponding to the level of Neutral Units. \\
-    \includesvg[height=15px]{\svgs/treasure.svg} & Roll a Treasure Die and gain the indicated bonus. \\
+	\includesvg[height=15px]{\svgs/treasure.svg} & Roll a Treasure Die and gain the indicated bonus (\includesvg[height=10px]{\svgs/resource_die.svg}, \includesvg[height=10px]{\svgs/2_treasure_die.svg}, \includesvg[height=10px]{\svgs/experience.svg}, or \includesvg[height=10px]{\svgs/artifact.svg}).\index{Treasure Die} \\
```

Comment:
```suggestion
    \includesvg[height=15px]{\svgs/treasure.svg} & Roll a Treasure Die and gain the indicated bonus (\includesvg[height=10px]{\svgs/resource_die.svg}, \includesvg[height=10px]{\svgs/2_treasure_die.svg}, \includesvg[height=10px]{\svgs/experience.svg}, or \includesvg[height=10px]{\svgs/artifact.svg}).\index{Treasure Die} \\
```

Reply:
Huh, the linter should have picked it up.

Reply:
Argh, it did, but it didn't fail xD
https://github.com/Heegu-sama/Homm3BG/actions/runs/8734963836/job/23966726435?pr=183